fiftyone/migrations/revisions/v1_4_0.py

```
@@ -0,0 +1,42 @@
+FiftyOne v1.4.0 revision.
+
+In previous versions we used the run_link field in delegated operations to
+store the path to the log file. This revision migrates the data from the 
+run_link field to the new log_path field allowing for the use of both fields 
+and a cleaner implementation.
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+from pymongo import UpdateOne
```

Ruff (0.8.2)

`pymongo.UpdateOne` imported but unused

Remove unused import: `pymongo.UpdateOne`

(F401)